.github/workflows/image-builder.yml

```
@@ -0,0 +1,37 @@
+name: Build and publish the container image
+
+on:
+  push:
+    branches:
+      - "main"
```

Follow the same sequence indentation style as below.
```suggestion
    - main
```